currency-rates/tools/logger.go

```diff
@@ -0,0 +1,42 @@
+package logger
```

Comment:
name of package and name of directory is different. This is violates GO way standarts.

Reply:
package name == directory name

Reply:
My bad. Thanks for noticing)